src/controller/data/System/SystemConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Data.System;
+
+public class SystemConfiguration
+{
+    public int Id { get; set; }
+    public string SystemName { get; set; } = "Controller";
+    public TimeZones TimeZone { get; set; } = TimeZones.UTC;
+    public int TimeBetweenDataUpdatesSeconds { get; set; } = 5;
+}
+
+public enum TimeZones
```

Comment:
Incomplete

Reply:
It is enough for now, will mart it as TODO though

Reply:
Can we please get a full timezone list? Come on...

Reply:
@NotSayo